test/test_site/site.json

```diff
@@ -1,14 +1,7 @@
 {
   "baseUrl": "/test_site",
-  "pages": [
-    {
-      "src": "index.md",
-      "title": "Hello World"
-    },
-    {
-      "src": "bugs/index.md",
-      "title": "Open Bugs"
-    }
+  "addressable": [
+    "**/index.md"
```

Comment:
Show example of `titlePrefix` and `keywords` if you're adding support in this PR.